thirdparty/README.md

```diff
@@ -130,25 +130,25 @@ Files extracted from upstream source:
 
 ## fonts
 
-- `NotoSans*.ttf`, `NotoNaskhArabicUI_*.ttf`:
+- `NotoSans*.woff2`, `NotoNaskhArabicUI_*.woff2`:
   * Upstream: https://github.com/googlefonts/noto-fonts
   * Version: v2017-10-24-phase3-second-cleanup
   * License: OFL-1.1
   * Comment: Use UI font variant if available, because it has tight vertical metrics and
     good for UI.
-- `JetBrainsMono_Regular.ttf`:
+- `JetBrainsMono_Regular.woff2`:
```

Comment:
I don't know if that makes any significant difference, but JetBrains seem to provide a woff2 version of the font themselves: https://github.com/JetBrains/JetBrainsMono/blob/master/fonts/webfonts/JetBrainsMono-Regular.woff2

Reply:
It's probably not, WOFF is just a compressed wrapper, all font structures should stay the same. Their version is 3 KB larger, but it probably depends only on a tool that was used to compress it.